content/applications/sales/crm/acquire_leads/generate_from_website.rst

```diff
@@ -1,78 +1,105 @@
-----------------------------
-
-First, go to the page where you want to put your contact form. In edit mode, drag
-the form builder onto the page, and you will be able to add all the fields
-you wish.
+To change the form to a specific sales channel, go to :menuselection:`Website --> Configuration -->
+Settings.` Under *Communication,* you can change which *Sales Team* or *Salesperson* is assigned
+to any incoming leads or opportunities from your Contact Form.
 
-.. image:: media/generate_from_website06.png
+.. image:: generate_from_website/contact-form-settings.png
    :align: center
+   :alt: Contact Form settings
 
-By default, any new contact form will send an email. You can switch to
-lead/opportunity generation in *Change Form Parameters*.
+Creating a custom Contact Form
+==============================
 
-.. note::
-   If the same visitor uses the contact form twice, the second
-   entry will be added to the first lead/opportunity in the chatter.
+Wanting more info from visitors using the Contact Form? Contact Forms can be customized for the
+specific information your team needs. They can generate multiple types of records in the
+system, such as emails, leads/opportunities, project tasks, helpdesk tickets, and more.
+
+The free *Form Builder* module can be installed from either the apps page or automatically when
+adding a form to a page in the Website Builder. Adding a whole new form can be great if you are
+soliciting certain information that needs filtering, such as suggestions, bug reports, etc.
+
+.. image:: generate_from_website/form-building-block.png
+   :align: center
+   :alt: Form Builder building blocks
+
+However, if the Contact Us Page's available predefined form options can work with some editing,
+editing these options is the fastest way to customize the form to fit your needs.
+
+Editing contact form fields
+---------------------------
+In edit mode, click on the field to change, and you can edit any of the following information for
+the specific field:
+
+- **Type**: Choose a custom field option or an existing field. Examples include phone, file
+  upload, language, etc.
+- **Input Type**: This determines the type of entry customers should input. Available options are
+  text, email, telephone, and URL.
+- **Input Placeholder**: Type in an example to guide users how to input information where
+  formatting is important, such as a phone number or email address.
+- **Label Name**: Type in the display name to show users what information is needed from them.
+- **Label Position**: Choose the way the label is aligned with the rest of the form. The label
+  can be hidden, above the field, to the far left of the field, or right adjusted and closer to the
+  field.
+- **Required**: Toggleable option for information that you absolutely need entered.
+- **Hidden**: Toggleable option to hide the given field for this form or not.
+- **Shown on Mobile**: Toggleable option to show the field to users on mobile devices.
```

Comment:
Small edit so that the wording matches the other bullet points in the list.

`**Shown on Mobile**: Toggle this option to show the field to users on mobile devices.`